### ISSUE TITLE
Disable retrocert auth page submit button upon submission

### ISSUE THREAD

**Issue by kalvinwang** [cagov/unemployment#635]
Currently it's possible to click the retrocert auth page submit button multiple times; if the response is 401 unauthorized (due to incorrect SSN, etc.) then the logs will show multiple 401s which pollutes the logs and makes it look like something is wrong. We should disable the submit button upon submission, until a response is received.

**Comment by idealisms** [comment]
I know there was some concern earlier about how it's confusing to disable buttons. We could also just not send multiple requests until the first one has returned.

It might also be good to have a timeout on the client side (30s? 60s?) in case the request is hanging for some unknown reason.

**Comment by kalvinwang** [comment]
Good point, we don't need need to visually disable the button. Hm, can we figure out what percentage of client-side requests take more than a couple seconds?...
